compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -326,7 +326,8 @@ internal actual class ComposeWindow : UIViewController {
         layer = ComposeLayer(
             layer = skiaLayer,
             platform = uiKitPlatform,
-            getTopLeftOffset = ::getTopLeftOffset,
+            // TODO: Review if it's needed for other users of ComposeLayer.jsNative.kt
```

Comment:
It is not needed, we can safely remove `getTopLeftOffset`

Reply:
Indeed, done